solutions/telemetry/console/producer/src/main/java/com/redhat/telemetry/ProducerApp.java

```diff
@@ -0,0 +1,45 @@
+package com.redhat.telemetry;
+
+import java.util.Properties;
+import java.util.Random;
+
+import org.apache.kafka.clients.CommonClientConfigs;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.config.SslConfigs;
+
+
+public class ProducerApp
+{
+    public static void main( String[] args )
+    {
+        Random random = new Random();
+        Producer<Void,Integer> producer = new KafkaProducer<>(configureProperties());
+
+        for (int i=0;i < 10; i++) {
```

Comment:
Perhaps we can do a infinite loop here and add a wait between loops

Reply:
@mabulgu you had some thoughts about infinite loops right? (while true)